src/tm_devices/commands/gen_1lh2st_msodpo/search.py

```diff
@@ -1117,42 +1078,6 @@ def polarity(self) -> SearchSearchItemTriggerATimeoutPolarity:
         """
         return self._polarity
 
-    @property
-    def source(self) -> SearchSearchItemTriggerATimeoutSource:
-        """Return the ``SEARCH:SEARCH<x>:TRIGger:A:TIMEOut:SOUrce`` command.
```

Comment:
What happened to this command?

Reply:
This command is available in MDO, MSO, LPDMSO and TekscopePC.

Reply:
So why was it removed from here?